docs/2021/buildsystem/build.md

```diff
@@ -40,14 +40,14 @@ Install CMake if you haven't already (minimum 3.10 required)
 
 | CMake Flags            | Description                  | Default     |
 | --- | --- | --- |
-| **-DCMAKE_INSTALL_PREFIX=\<path\>** | Sets the install prefix. | `/usr/local` |
-| **-DAGENTS=\"agent1;agent2...\"** | Only configure these agents. | ALL AGENTS  |
-| **-DOFFLINE=\<ON/OFF\>** | Controls vendor generation, ON=NO  | **OFF**     |
-| **-DCMAKE_BUILD_TYPE=\<type\>** | -   `Debug`, `Release`, `RelWithDebInfo`,`MinSizeRel`     | `Debug` |
-| **-DTESTING=\<ON/OFF\>** | Controls testing config generation    | > **OFF**   |
-| **-DMONOPACK=\<ON/OFF\>** | Package adj2nest and ununpack seperately        | **OFF**     |
+| **-DCMAKE_INSTALL_PREFIX=<path\>** | Sets the install prefix. | `/usr/local` |
+| **-DAGENTS="agent1;agent2..."** | Only configure these agents. | ALL AGENTS  |
+| **-DOFFLINE=<ON/OFF\>** | Controls vendor generation, ON=NO  | **OFF**     |
+| **-DCMAKE_BUILD_TYPE=<type\>** | -   `Debug`, `Release`, `RelWithDebInfo`,`MinSizeRel`     | `Debug` |
+| **-DTESTING=<ON/OFF\>** | Controls testing config generation    | **OFF**   |
+| **-DMONOPACK=<ON/OFF\>** | Package adj2nest and ununpack seperately        | **OFF**     |
```

Comment:
Hi @its-sushant, are you sure this will be rendered as expected, since angled brackets are considered as HTML tags, we must escape them if we want to display them as it is. You converted them to HTML tags. 

Also a single closing angle bracket is used for quotation. and that is intentional. Please build your pull request branch and make sure that it renders right.

Reply:
Hey @avinal , These changes are here in the pull request due to wrong rebase done by me. There is one commit by gaurav here of which you are referring to. I will try and drop the commit or rebase it again. Thanks for noticing it anyways. :D